priv/perf/apps/load_testing/lib/application.ex

```diff
@@ -15,7 +17,9 @@ defmodule LoadTest.Application do
         max_connections: max_connections
       )
 
-    {:ok, self()}
+    children = [{LoadTesting.Utils.NonceTracker, %{}}, {LoadTesting.Utils.Faucet, nil}]
+
+    Supervisor.start_link(children, name: LoadTesting.Supervisor, strategy: :one_for_one)
```

Comment:
I think for NonceTracker it is fine to be supervised as it would ask eth client for current nonce once restarted.
Not sure whether Faucet logic is designed to be restart-able though. In the case of faucet is not set from config, it would actually re-generate a new faucet. Can you double check and confirm that we want to supervise Faucet or not?

Reply:
if this is the case, the PR shouldn't be merged before we figure this one out. happy to help figure out the strategy or supervision hierarchy. ping me!

Reply:
I don't really think there is need to supervise any of the two (`NonceTracker`, `Faucet`) for this test.... those Genservers are really just some place we want/need to hold temporary states in memory. Although it *might* be more stable but without fully thought over and well designed yet personally would prefer to just fail the test if the process ever break.

But this is from a elixir n00bie point of view.....